content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
 
+### Collaborative writing and reviewing
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for more technical writing [@url:https://www.overleaf.com/ @url:https://www.authorea.com/].
+`TODO: survey other existing options, see if there are better references`
+These platforms typically offer version control and other features to support multi-author editing.
+`TODO: some use git in the back end, is that relevant?`
+However, their critical limitation for open writing is that they give all authors permission to modify all parts of the manuscript.
+This makes it difficult to maintain consistent content and tone and is susceptible to malicious edits.
+`TODO: are wikis, especially Wikipedia, relevant as an alternative approach?`
+
+We adopted standard software development strategies in order to enable any contributor to edit any part of the manuscript but enforce discussion and review of all proposed changes.
+The GitHub platform provided support for organizing and editing the manuscript.
+We used GitHub _issues_ for organization, opening a new issue for each paper under consideration.
+Within the issue, contributors summarized the research, discussed it (sometimes with the original authors), and assessed its relevance to the review.
+Issues also served as an open to-do list and a forum for debating the main message, themes, and topics of the review.
+
+GitHub and the underlying git version control system also structured the writing process.
+An official master version could be _forked_ by individual contributors.
+A contributor could then add and revise any files, grouping these changes into _commits_.
+When the changes were ready to be reviewed, the series of commits could be submitted as a _pull request_ through GitHub, which notified other authors of the pending changes.
+GitHub's review interface allows anyone to comment on the changes, globally or at specific lines, asking questions or requesting modifications from the author.
+Once all requested modifications were made, the manuscript owners, a subset of authors with elevated GitHub permissions, could formally approve the pull request and merge the changes into the official master version.
```

Comment:
I don't think "owners" is the right term. In GitHub parlance, those with "write" or "admin" access can merge PRs. In software, these individuals are generally called maintainers. GitHub uses the "Member" (or "Owner") label next to individuals names who fit this criteria.